### PR TITLE
Add Cm

### PR DESCRIPTION
**Pull request by rhyspoulton**
Cm is a C library for multitask communication

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fcarbogn** [comment]
I am willing to be listed as maintainer

File: recipes/Cm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "cm" %}
+{% set version = "9.0.0" %}
+{% set versions = version.split('.') %}
```
Comment (duncanmmacleod) [inline recipes/Cm/meta.yaml:3]: This line is unused and should be removed.

File: recipes/Cm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - pkg-config --print-errors --modversion cm
+
+about:
+  home: https://git.ligo.org/virgo/virgoapp/Cm/
+  dev_url: https://git.ligo.org/virgo/virgoApp/Cm/
+  license: BSD-4-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: A multitask communication package
+  description: |
```
Comment (duncanmmacleod) [inline recipes/Cm/meta.yaml:43]: This probably should be empty, either include a `description`, or remove the empty variable.

File: recipes/Cm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: ffcd65f5ff779db07bc2fbe982da6277ffa495aa6d66300c3194bd4a344311a4
+
+build:
+  number: 0
+  skip: True  # [win or osx]
```
Comment (duncanmmacleod) [inline recipes/Cm/meta.yaml:15]: Once the feedstock is created, I can help configure builds for these platforms, presuming the package builds on them in the first place.

File: recipes/Cm/meta.yaml (hunk 1 above)
Comment (rhyspoulton) [reply recipes/Cm/meta.yaml]: I have removed this line.

File: recipes/Cm/meta.yaml (hunk 2 above)
Comment (rhyspoulton) [reply recipes/Cm/meta.yaml:42]: I have added a short description of the package to this line.

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this recipe is ready for review, thanks.

File: recipes/Cm/meta.yaml (hunk 3 above)
Comment (rhyspoulton) [reply recipes/Cm/meta.yaml:14]: Unfortunately, the package doesn't build on OSX and windows as you can see from the checks [here](https://github.com/conda-forge/staged-recipes/runs/1878807711), due to combability issues on OSX and required libraries not available on Windows. But the developer is currently attempting to port the code to OSX and possibly windows.

**Comment by rhyspoulton** [comment]
Thank you for your help @duncanmmacleod.

**Pull request merged**